app/models/podcast_episode.rb

```diff
@@ -15,7 +15,7 @@ class PodcastEpisode < ApplicationRecord
   delegate :published, to: :podcast
 
   belongs_to :podcast
-  has_many :comments, as: :commentable, inverse_of: :commentable
+  has_many :comments, as: :commentable, inverse_of: :commentable, dependent: :nullify
```

Comment:
**Question**: I've never really used nullify before, but I read up on it and wanted to understand more. When a Podcast is deleted we destroy the depending podcast_episodes, so why do we nullify the comments instead of destroying them?

Reply:
A little while ago we determined that comments really belong to the user that wrote them and that they might contain pertinent information even without the context of the original commentable so we decided to keep them even when their commentable is removed. 

On a separate note, going forward comments with commentable will have their commentable ID set to null, should we backfill null values for those comments that have already had their commentable removed?

Reply:
@mstruve I haven't found any related to podcast episodes, see the third clause here https://dev.to/admin/blazer/queries/194-orphan-podcast-episodes

I agree though :) There are if we check the articles table, but I left `Article`, `Organization` and `User` to the end of this "cleanup sprint" as they are the most connected: https://dev.to/admin/blazer/queries/196-comments-with-an-invalid-article-id - 8,916 comments with an invalid article id